test/fixtures/test-output.txt

```diff
@@ -1,9 +1,5 @@
 🍝
 You should add some tests.
 Installing 3 dependencies.
-"prelude" already installed
-"console" already installed
-"effect" already installed
 Installation complete.
 Build succeeded.
-Tests succeeded.
```

Comment:
I don't think this was supposed to be removed... At least, it would fail the test on mine if it did.

Reply:
Yup
```
Output doesn't match fixture!


---- Expected ------
🍝
You should add some tests.
Installing 3 dependencies.
"prelude" already installed
"console" already installed
"effect" already installed
Installation complete.
Build succeeded.
Tests succeeded.

---- Actual --------
🍝
You should add some tests.
Installing 3 dependencies.
Installation complete.
Build succeeded.
Tests succeeded.

-----------
```

Reply:
Sorry it slipped in :)